assets/scripts/app/focus.js

```diff
@@ -44,18 +42,8 @@ export function onWindowFocus () {
   saveSettingsLocally()
 }
 
-function onWindowBlur () {
-  if (abortEverything) {
```

Comment:
That if-statement doesn't exist in new version. Is this planned? We want to trigger just `hideAllMenus` on `blur` event?